history-of-quakers/modernized/23-chapter-21.adoc

```diff
@@ -1246,12 +1364,15 @@ as may secure it to our posterity in after-times; and while we live,
 it shall be our endeavor, through God`'s grace,
 to demean ourselves as in conscience to God, and duty to the king, we are obliged.
 
-His peaceable, loving,
+[.signed-section-signature]
```

Comment:
Not sure if this should be [sss].

Reply:
It's borderline, but I think I kind of like it how you did it here, as a signature. Let's go with it.